README.md

```diff
@@ -147,6 +147,91 @@ expect(cat.sound(), "Purr");
 //using real object
 expect(cat.lives, 9);   
 ```
+
+## Strong mode compliance
+
+Unfortunately, the use of the arg matchers in mock method calls (like `cat.eatFood(any)`)
+violates the [Strong mode] type system. Specifically, if the method signature of a mocked
+method has a parameter with a parameterized type (like `List<int>`), then passing `any` or
```

Comment:
Any implicit or explicit cast might fail in DDC.  Similarly, any implicit or explicit cast might fail in regular Dart checked mode.  In general, for simple types (like String), if the cast passed in checked mode, it will pass in DDC.  For complex types (like `List<String>`), DDC may throw a type error where checked mode wouldn't have.  This is because the checked mode check is unsound for composite types like `List<String>` (a `List<dynamic>` will be allowed through), and so DDC strengthens the cast.  Since these are places where code that worked in checked mode may fail in DDC, we emit a special  warnings for these casts that might fail in DDC where the would not have in checked mode.